platform/test/spec/testActivityManagerSpec.js

```diff
@@ -6,91 +6,67 @@ import { ActivityConfigValidator } from "../../src/ActivityConfigValidator.js";
 import { utility } from "../../src/Utility.js";
 import { ACTIVITY_2PANELS_1ACTION } from "../resources/TestActivityFiles.js";
 import { TOOL_1PANELDEF_1FUNCTION } from "../resources/TestToolFiles.js";
-import { configObjectEquals, configToString } from "../resources/TestUtility.js";
+import { configObjectEquals } from "../resources/TestUtility.js";
 
 
 describe("ActivityManager", () => {
 
-    it("can be created", () => {
+    describe("constructor", () => {
         // Setup
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
-            
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
-
-        // Check the expected results
-        expect(am).toBeInstanceOf(ActivityManager);
-    })
-
-    it("constructor -  initialises the configValidator property", () => {
-        // Setup
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
+        const ACTIVITY_URL="test://a.url";
+        let am;
+        let fileh;
+        let refPanelDef;
+        let spySearch;
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+        beforeEach( () => {
 
-        // Check the expected results
-        expect(am.configValidator).toBeInstanceOf(ActivityConfigValidator);
-    })        
+            fileh = new FileHandler("test://th.url") 
+            refPanelDef = () => {};
 
-    it("constructor -  initialises the accessPanelDef property using param panelDefAccessor", () => {
-        // Setup
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
+            spySearch = spyOn( utility, "getWindowLocationSearch");
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+            // Call the target object
+            am = new ActivityManager(refPanelDef, fileh);
```

Comment:
Should this be in each test (there is at least one test that has to call this again anyway...)?

Reply:
Yes it does make each test uniform fixed in 023439982f88091a37a89567f047fcb5495458bd